src/main/java/seedu/address/model/event/fields/Description.java

```diff
@@ -0,0 +1,28 @@
+package seedu.address.model.event.fields;
+
+/**
+ * Represents an Events's Description in the Calendar.
+ * Guarantees: immutable; is valid as declared in {@link #isValidDescription(String)}
+ */
+public class Description {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Description should only contain alphanumeric characters and spaces, "
+                    + "and it should not be blank";
+
+    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
```

Comment:
Could consider loosening this constraint, for cases like `Somebody's birthday` (apostrophe is not alphanumeric).

Reply:
I think we can make the description not have a validation regex at all. So we just accept anything.

Reply:
okay but i think i am gonna have a constraint where the description cant be blank but i think it will be handled in the requireNonNull so not sure if i should add it